docs/pages/user-docs/search.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Search
+parent: User Guide
+nav_order: 620
+---
+# :mag_right: Search
+You can search for names, abbreviations, locations or departments. The search results display contact information your colleagues and details of their employment. 
+
+To contact another user directly from the result list you click the ![envelope icon](media/envelope.png) next to the name. This will open your email program. The same logic applies for **phone calls** after clicking the displayed number of a user. 
+
+If you search for a unique name or abbreviation the search will open the relevant profile on its own. Otherwise you have to select the right entry from the result list. 
+
+![screenshot of search results](media/searchresults.PNG)
+## :wrench: Configure search results
+It is possible to reduce the search results by hiding selected attributes. Select  ![gear icon](media/gearicon.PNG) to open the table columns configuration. Use Drag & Drop to sort the attributes in "displayed" and "not displayed".
+
+![screenshot of table columns configuation](media/configuretable.PNG)
+## :small_red_triangle_down: Filter
+Every column of the result list can be used as a filter. To do so you need to type in the relevant attribute and define its value with ": value X". Confirm your filter with Enter and go on with typing your search request. 
+
+Example: You want to find Thomas from Munich 
+
+1. ![city: munich](media/filter1.PNG) 
+1. **Enter** 
+1. ![established city filter + "Thomas"](media/filter2.PNG)
+
+When you've opened a profile - whether it's yours or another users - you can use all clickable attributes (bright colored) as filters, too. 
+
+**Caution!** You can only use filters that are configurate as "displayed" in the filter settings (see above).
```

Comment:
That's not right, you can filter for everything.

Reply:
well, when I wrote the guide it wasn't possible. But if you say so I'll change that

Reply:
You mean a different thing. You mean that it's not possible to search for columns which are not shown, but it was always possible to use the filter by clicking on it.